src/components/connect.js

```diff
@@ -153,11 +156,23 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
 
       refetchDatum(prop, mapping) {
         const startedAt = new Date()
+        const isRefresh = mapping.refreshInterval > 0
 
         if (this.state.refreshTimeouts[prop]) {
           window.clearTimeout(this.state.refreshTimeouts[prop])
         }
 
+        // do not refresh while tab is hidden and refreshHidden is unset
+        if (isRefresh && !mapping.refreshHidden && visibility.hidden()) {
```

Comment:
I understand and agree but seems to be that `this.state.mappings[prop]` doesn't have `fulfilled ` property. I need to dig deeper to understand the code and check for it. Could you help here?